### DISCUSSION TITLE
Want changing in header of website

### DISCUSSION THREAD

**Discussion by chasdse** [moja-global/community-website#435] [General]
Can I change the header of my website as a beginner, and if so, how can I do it[?](https://coolnamesfinder.com/) I'd like to make this modification, and I have a specific website in mind that I'd like to share for guidance. Experts, could you please provide step-by-step instructions or advice on how to accomplish this?

**Comment by locksmith786** [comment]
Yes. If you have access to the website’s source code, you can change the header even as a beginner.

A typical process is:

Back up your website before making changes.
Find the header file. Depending on the platform, it may be header.php[,](https://locksmithhighlandranch.com/) header.html, or a component such as Header.jsx.
Locate the existing HTML for the navigation/logo area.
Modify the HTML to add, remove, or rearrange elements.
Use CSS to control its appearance, for example:
```
header {
    padding: 20px;
    background: #fff;
}

nav {
    display: flex;
    align-items: center;
    justify-content: space-between;
}

```Save the changes and test the header on both desktop and mobile.
